sdks/go/pkg/beam/runners/dataflow/dataflowlib/job.go

```diff
@@ -128,6 +129,15 @@ func Translate(ctx context.Context, p *pipepb.Pipeline, opts *JobOptions, worker
 		experiments = append(experiments, "use_staged_dataflow_worker_jar")
 	}
 
+	for _, url := range opts.ArtifactURLs {
+		name := url[strings.LastIndexAny(url, "/")+1:]
+		jar := &df.Package{
```

Comment:
nit: The artifacts aren't necessarily going to be Jars, as the expansions aren't necessarily from Java. Maybe pkg which is reasonably generic?

Reply:
Good point, done.